prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -117,7 +117,7 @@ public async Task SendSiteReviewedNotificationAsync(int siteId, string note)
 
         public async Task SendRemoteUsersUpdatedAsync(CommunitySite site)
         {
-            var downloadUrl = await _emailDocumentService.GetBusinessLicenceDownloadLink(site.Id);
+            var downloadUrl = await _emailDocumentService.GetBusinessLicenceDownloadLink(site.BusinessLicence.Id);
```

Comment:
Question:  Is the erroneous behavior addressed by this PR that either the business licence would not be downloadable or that the wrong business licence would be linked to?

![image](https://user-images.githubusercontent.com/7586665/211362812-2efbe526-0888-4563-a29c-188c7d77eb74.png)

Reply:
The only issue this PR fixing is that the SendRemoteUsersUpdatedAsync was throwing exception and the targeted email was failed to send out. I am not aware of other issues.

Reply:
Obviously this fix makes sense, but how do you reproduce the error ... is it only if the value of the `site.Id` isn't in the `Id` field of the `BusinessLicence` table?